internal/interceptor.go

```diff
@@ -239,7 +239,8 @@ type WorkflowOutboundInterceptor interface {
 	// SetQueryHandler intercepts workflow.SetQueryHandler.
 	SetQueryHandler(ctx Context, queryType string, handler interface{}) error
 
-	SetUpdateHandler(ctx Context, updateName string, handler interface{}, opts UpdateOptions) error
+	// SetUpdateHandler intercepts workflow.SetUpdateHandler.
```

Comment:
Let's toss experimental warnings on this stuff too

Reply:
Yes, and added comments too - I missed these initially.

Reply:
Still missing at least a `NOTE: Experimental` or something on these.

Reply:
Still missing at least a `NOTE: Experimental` or something on these.